code/game/area/areas/shuttles.dm

```diff
@@ -159,11 +159,11 @@
 
 /area/shuttle/escape/luxury
 	name = "Luxurious Emergency Shuttle"
-	noteleport = TRUE
+	teleport_restriction = TELEPORT_ALLOW_ABDUCTORS
 
 /area/shuttle/escape/arena
 	name = "The Arena"
-	noteleport = TRUE
+	teleport_restriction = TELEPORT_ALLOW_NONE
```

Comment:
What's up with this inconsistency?

Reply:
Arena is in another dimension, luxury shuttle isnt.

Reply:
That would be `/area/shuttle_arena`